packages/framework/src/Model/Blog/Category/BlogCategoryVisibilityRepository.php

```diff
@@ -109,8 +109,15 @@ protected function refreshBlogCategoriesVisibilityOnDomainAndLevel(DomainConfig
                     bct.name IS NOT NULL
                     AND
                     parent_bcd.visible = TRUE
+                    AND
+                    (
+                        SELECT COUNT(ba.id) 
+                        FROM blog_articles AS ba 
+                        JOIN blog_article_domains AS bad ON bad.blog_article_id = ba.id AND bad.domain_id = :domainId AND bad.visible = TRUE
+                        JOIN blog_article_blog_category_domains AS babcd ON babcd.blog_article_id = ba.id AND babcd.blog_category_id = bcd.blog_category_id AND babcd.domain_id = 1
```

Comment:
When I create a new blog category and article in the second domain, the category is still hidden. I guess it is caused by `babcd.domain_id = 1` on this line.